internal/github/owners.go

```diff
@@ -0,0 +1,40 @@
+package github
+
+import "math/rand"
+
+var reviewers = map[string]string{
+	"Brett Thurber":         "bthurber",
+	"Chris Procter":         "chr15p",
+	"Enrique Belarte Luque": "enriquebelarte",
+	"Ezequiel Russo":        "erusso7",
+	"Fabien Dupont":         "fabiendupont",
+	"Michail Resvanis":      "mresvanis",
+	"Pablo Iranzo Gómez":    "iranzo",
+	"Quentin Barrand":       "qbarrand",
+	"Yevgeny Shnaidman":     "yevgeny-shnaidman",
+	"Yoni Bettan":           "ybettan",
```

Comment:
I don't think we should hardcode this.

Reply:
I didn't like it either but how will we get:
* the github login from the commit (non related to github)?
* a default reviewer in case the original author isn't from the team (or from Redhat)?

Reply:
Replaced it with API calls to the u/s OWNERS file.